Merge release/19.1 into trunk (19.1-rc-1 code freeze)

AliSoftware opened the pull request:
Code freeze and release of [19.1-rc-1](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/19.1-rc-1).

- [x] New empty entry created in `RELEASE-NOTES.txt` for next iteration.
- [x] `{jetpack_}metadata/release_notes.txt` extracted for WordPress and Jetpack.
- [x] `WordPress/jetpack_metadata/release_notes.txt` audited to remove any item not applicable for Jetpack.
- [x] Internal dependencies (FluxC, Login. Stories. About…) bumped to a new stable version in `build.gradle` if necessary.
- [x] `WordPress/src/main/res/values/strings.xml` updated with strings from binary dependencies.
- [x] New strings frozen/copied into `fastlane/resources/values/strings.xml`.
- [x] Version bumped in `version.properties`

AliSoftware committed:
Bump version number

AliSoftware committed:
Clean up release files.

AliSoftware committed:
Update draft release notes for 19.1.

AliSoftware committed:
Update draft release notes for Jetpack 19.1.

AliSoftware committed:
Release Notes: add new section for next version (19.2)

AliSoftware committed:
Update strings for translation

AliSoftware committed:
Send strings to translation.

AliSoftware committed:
Update FluxC to 1.34.0

mokagio reviewed (approved):
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] Version updates in `version.properties`
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] `strings.xml` updates via automation merging values from libraries
- [x] Updated `ext` versions

---

FWIW, I removed the "into `<branch name>`" from my code freeze merge PR because, without the `develop`-`trunk` distinction, this kind of PRs should always and only be into `trunk`. 🤷

AliSoftware commented:
> FWIW, I removed the "into `<branch name>`" from my code freeze merge PR because, without the `develop`-`trunk` distinction, this kind of PRs should always and only be into `trunk`. 🤷

Yeah I removed it from my similar PRs last week for that same reason 👍  but… old habits die hard apparently, as it naturally came back without me realizing when I instinctively typed the PR title on this one today 😄
